### PR TITLE
🧹 remove v8 platform deprecations

### PR DESCRIPTION
**Pull request by arlimus**
Release is replaced by Version, so remove it.

### CODE REVIEW

File: motor/platform/platform.proto [hunk 1]
```diff
@@ -7,8 +7,6 @@ option go_package = "go.mondoo.com/cnquery/motor/platform";
 
 message Platform {
   string name = 1;
-  // FIXME: remove in v8.0 vv
```
Comment (arlimus) [inline motor/platform/platform.proto:10]: removed due to this indicator, the rest is cleanup

File: mql/mql_test.go [hunk 2]
```diff
@@ -63,12 +67,12 @@ func TestMqlSimple(t *testing.T) {
 		query     string
 		assertion interface{}
 	}{
-		{"platform.name", "arch"},
-		{"platform { name release }", map[string]interface{}{
-			"name":    "arch",
-			"release": "",
+		{"asset.platform", "arch"},
```
Comment (arlimus) [inline mql/mql_test.go:70]: we have been switching from platform to asset in v7; this is coming late

File: mql/mql_test.go [hunk 3]
```diff
@@ -63,12 +67,12 @@ func TestMqlSimple(t *testing.T) {
...
+		{"asset { platform version }", map[string]interface{}{
+			"platform": "arch",
+			"version":  "",
 		}},
-		{"users.list { name uid }", []interface{}{
+		{"users { name uid }", []interface{}{
```
Comment (arlimus) [inline mql/mql_test.go:75]: another leftover v7 change that was missing

File: motor/platform/platform.proto (hunk 1 above)
Comment (jaym) [reply motor/platform/platform.proto:10]: does this get stored as proto somewhere. Do you need to mark 2 as unusable?

**Review by jaym — APPROVED** [review]

**Pull request merged**